src/client/tile.cpp

```diff
@@ -1134,7 +1141,10 @@ video::IImage * Align2Npot2(video::IImage * image,
 	core::dimension2d<u32> dim = image->getDimension();
 
 	std::string extensions = (char*) glGetString(GL_EXTENSIONS);
-	if (extensions.find("GL_OES_texture_npot") != std::string::npos) {
+
+	// Only GLES2 is trusted to correctly report npot support
```

Comment:
so in case an Android device only support gles1 we assume npot is needed, correct?